helpers/normalize-responses.js

```diff
@@ -148,8 +164,11 @@ module.exports = function normalizeResponses (configuredResponses, exits){
     if (exitDef.responseType === 'redirect') {
       // Note that we tolerate the absense of an outputExample, since a redirect is assumed to always be a string.
       if (!_.isUndefined(outputExample) && !_.isString(outputExample)) {
-        throw new Error(util.format('`machine-as-action` cannot configure exit "%s" to redirect.  The redirect URL is based on the return value from the exit, so the exit\'s `example` must be a string.  But instead, it\'s: ', exitCodeName,util.inspect(outputExample, false, null)));
-      }
+        throw flaverr(
+          'E_INVALID_RES_METADATA_IN_EXIT_DEF',
+          new Error(util.format('Cannot configure exit "%s" to redirect.  The redirect URL is based on the return value from the exit, so the exit\'s `outputExample` must be a string.  But instead, it is: ', exitCodeName,util.inspect(outputExample, false, null)))
```

Comment:
@mikermcneil In this error (and the next two), you replaced `machine-as-action` with just capitalized `Cannot`, but you left `machine-as-action` in the errors above this one.  Not sure if that was intentional or not.